app/assets/javascripts/admin/addon/components/admin-branding-fonts-form.gjs

```diff
@@ -0,0 +1,286 @@
+import Component from "@glimmer/component";
+import { cached, tracked } from "@glimmer/tracking";
+import { fn } from "@ember/helper";
+import { action } from "@ember/object";
+import { service } from "@ember/service";
+import { classify, decamelize, underscore } from "@ember/string";
+import DButton from "discourse/components/d-button";
+import Form from "discourse/components/form";
+import UpdateDefaultTextSize from "discourse/components/modal/update-default-text-size";
+import concatClass from "discourse/helpers/concat-class";
+import { ajax } from "discourse/lib/ajax";
+import { bind } from "discourse/lib/decorators";
+import { i18n } from "discourse-i18n";
+import eq from "truth-helpers/helpers/eq";
+
+const FONTS = [
+  "Helvetica",
+  "Inter",
+  "Lato",
+  "Montserrat",
+  "OpenSans",
+  "Poppins",
+  "Roboto",
+  "Merriweather",
+  "Mukta",
+];
+const MORE_FONTS = [
+  "Arial",
+  "Lora",
+  "NotoSans",
+  "NotoSansJP",
+  "Nunito",
+  "Oswald",
+  "Oxanium",
+  "PT Sans",
+  "PlayfairDisplay",
+  "Raleway",
+  "RobotoCondensed",
+  "RobotoMono",
+  "RobotoSlab",
+  "SourceSansPro",
+  "System",
+  "Ubuntu",
+];
```

Comment:
I think it would be useful to think about what happens when a new font is added to `DiscourseFonts`. How will we keep this in sync with the changes?